xunit-autolabeler-v2/ast_parser/core/constants.py

```diff
@@ -17,7 +17,7 @@
 from frozendict import frozendict
 
 
-REGION_TAG_ONLY_REGEX = re.compile(r'(?<=\s)+\w+(?=])')
+REGION_TAG_ONLY_REGEX = re.compile(r'(?<=\s)+(\w|-)+(?=])')
```

Comment:
This regex looks very crucial to the project, what about having a parameterized test for it to make sure it's accurate?